prow/cmd/gerrit-crier/main.go

```diff
@@ -133,7 +134,9 @@ func main() {
 			&workqueue.BucketRateLimiter{Limiter: rate.NewLimiter(rate.Limit(10), 100)},
 		))
 
-	controller := crier.NewController(client, queue, prowjobInformerFactory.Prow().V1().ProwJobs())
+	gerritReporter := reporter.NewReporter()
+
+	controller := crier.NewController(client, queue, prowjobInformerFactory.Prow().V1().ProwJobs(), gerritReporter)
```

Comment:
To share the caches the controllers need to be in-process -- see upstream controller-manager codebase.

Reply:
ah... didn't really try with multiple criers, will follow up this.